roles/awscli/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "check for awscli"
+  stat: path=/usr/local/bin/aws
+  register: awscli_stats
+
+- name: "fetch awscli"
+  get_url:
+    url: https://s3.amazonaws.com/aws-cli/awscli-bundle.zip
```

Comment:
You might want to provide a sha here. With that, there's a bonus that ansible is smart about re-doing the download and then you can ditch the first stanza in this play and switch off of .changed from this stanza instead.